### PR TITLE
Chinese linguistics

### PR DESCRIPTION
**Pull request by kuipertan**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by bratseth** [comment]
Great! Currently the jar file is just checked in and not used so this doesn't build.

If you add this dependency:

```
    <dependency>
      <groupId>com.huaban</groupId>
      <artifactId>jieba-analysis</artifactId>
      <version>1.0.2</version>
    </dependency>

```

that is solved, but that gives 

JiebaTokenizer.java:[64,31] cannot find symbol
[ERROR]   symbol:   method initUserDict(java.lang.String[])
[ERROR]   location: variable segmenter of type com.huaban.analysis.jieba.JiebaSegmenter
[ERROR] 

Perhaps because this is 1.02 instead of 1.0.3? There is no 1.0.3 in any public repo.
I think you either need to make this work with 1.0.2, or else follow https://devcenter.heroku.com/articles/local-maven-dependencies to install the dependency from a local file. The first is preferable of course.

Lastly, the reason this PR doesn't show the build failure because this module isn't included in the root pom. 
Please add 
`    <module>vespa-chinese-linguistics</module>
`
to it.

**Comment by kuipertan** [comment]
Sorry, I forgot to add the detail description.

I just added this description in the Readme:

mvn install:install-file -Dfile=./libs/jieba-analysis-1.0.3-SNAPSHOT.jar -DgroupId=com.huaban.analysis -DartifactId=jieba -Dversion=1.0.3-SNAPSHOT -Dpackaging=jar

Besides, "jieba" is referenced in Readme, I build it byself, and import the version 1.0.3 of jieba-analysis to mvn repos.

**Comment by bratseth** [comment]
mvn install should just work, but I can add that step to the pom if you like.

But could you answer this:
- are you sure you can't use 1.0.2 so we could avoid this?
- do you know why 1.0.3 isn't available?

File: vespa-chinese-linguistics/src/main/java/com/qihoo/language/JiebaTokenizer.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (bratseth) [inline vespa-chinese-linguistics/src/main/java/com/qihoo/language/JiebaTokenizer.java:1]: Please change to your copyright: Copyright 2020 Qihoo Corporation.

**Comment by kuipertan** [comment]
jieba-1.0.2 does not contain  function initUserDict which is used to load customed dict.     See [https://github.com/huaban/jieba-analysis/commit/387e21bc638c1ea4e8a05b44961ff496d411a93b#diff-b2fc2e9f593a80f0ca88bba6518b75ee001a2f8e1bcd128a313ad25e104b9fc8](url)  .   That is the latest code, but no mave repo containning 1.0.3.  So I  attachted it in dir libs.

File: vespa-chinese-linguistics/src/main/java/com/qihoo/language/JiebaTokenizer.java (hunk 1 above)
Comment (kuipertan) [reply vespa-chinese-linguistics/src/main/java/com/qihoo/language/JiebaTokenizer.java]: Changed.

**Comment by bratseth** [comment]
Thanks. I'll merge and then change to use a local repo.

**Pull request merged**